src/lib/mina.ts

```diff
@@ -677,22 +677,34 @@ LocalBlockchain satisfies (...args: any) => Mina;
  * Represents the Mina blockchain running on a real network
  */
 function Network(graphqlEndpoint: string): Mina;
-function Network(graphqlEndpoints: {
+function Network(endpoints: {
+  mina: string | string[];
+  accountsManager?: string;
+}): Mina;
+function Network(endpoints: {
   mina: string | string[];
   archive: string | string[];
+  accountsManager?: string;
```

Comment:
same comment as before: I'd advocate for making this name even more explicit to avoid confusing people. Like `lightnetAccountManager`

Also, can you add an explanation of the accounts manager argument to the doccomment on this function?